charts/neo4j-internal/values.preprod.yaml

```diff
@@ -56,7 +56,7 @@ neo4j-standalone:
         command: ['/data/docker-neo4j/scripts/dataset-download.sh', '-p', '/data']
         envFrom:
           - secretRef:
-              name: neo4j-preprod-nexus-dataset # imports NEXUS_TOKEN
+              name: neo4j-nexus-dataset-secret # imports NEXUS_TOKEN
```

Comment:
I confirm that both secret are existing and with correct values on dev and preprod namespace

Reply:
and i also confirm that kyverno policy worked and it duplicate secret from preprod to dev